wandb/sdk/internal/internal_api.py

```diff
@@ -3284,6 +3295,65 @@ def create_artifact_type(
         _id: Optional[str] = response["createArtifactType"]["artifactType"]["id"]
         return _id
 
+    def server_create_artifact_introspection(self) -> List:
```

Comment:
Nit: let's cache the query result, similar to what `server_use_artifact_input_introspection` does.